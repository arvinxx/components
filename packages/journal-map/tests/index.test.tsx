import React from 'react';
import { render } from '@testing-library/react';

import JournalMap from '@arvinxu/journal-map';

describe('JournalMap', () => {
  it('默认状态', () => {
    const { container } = render(
      <JournalMap data={{ actions: {}, steps: [] }} />,
    );
    expect(container).toMatchSnapshot();
  });
});
