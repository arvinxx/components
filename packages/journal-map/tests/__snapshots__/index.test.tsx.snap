// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JournalMap 默认状态 1`] = `
<div>
  <div
    class="avx-journal-map-container"
  >
    <div
      class="avx-journal-map-content"
    >
      <div
        class="avx-journal-map-flow-container"
      >
        <div
          style="width: 40px;"
        >
          <div
            class="avx-journal-map-stage"
            style="height: 48px;"
          >
            <div
              class="avx-journal-map-stage-text"
            >
              阶段
            </div>
          </div>
          <div
            style="margin-top: 24px;"
          >
            <div
              class="avx-journal-map-stage"
              style="height: 120px;"
            >
              <div
                class="avx-journal-map-stage-text"
              >
                用户行为
              </div>
            </div>
          </div>
        </div>
        <div
          class="avx-journal-map-flow-content"
        />
      </div>
      <div
        class="avx-journal-map-chart"
      >
        <div
          style="width: 40px;"
        >
          <div
            class="avx-journal-map-stage"
            style="height: 300px;"
          >
            <div
              class="avx-journal-map-stage-text"
            >
              体验情绪
            </div>
          </div>
        </div>
        <div
          style="margin-top: 2px; width: 100%;"
        >
          <div
            data-chart-source-type="G2Plot"
            size-sensor-id="1"
            style="height: inherit;"
          >
            <div
              style="position:relative;"
            >
              <canvas
                height="300"
                style="width: 400px; height: 300px; display: inline-block; vertical-align: middle;"
                width="400"
              />
            </div>
            <object
              class="size-sensor-object"
              data="about:blank"
              style="display: block; position: absolute; top: 0px; left: 0px; height: 100%; width: 100%; overflow: hidden; pointer-events: none; z-index: -1; opacity: 0;"
              tabindex="-1"
              type="text/html"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
